addons/info/src/components/markdown/text.js

```diff
@@ -42,8 +42,8 @@ export function A(props) {
   const style = {
     color: '#3498db',
   };
-  return <a href={this.props.href} style={style}>{props.children}</a>;
+  return <a href={props.href} style={style}>{props.children}</a>;
```

Comment:
Agree, I always put external links here, and there is already `@storybook/addon-links` solution for linking between stories at same site.